### ISSUE TITLE
Home page seems bare

### ISSUE THREAD

**Issue by dhasial** [howtostartacoop/howtostartacoop#1]
There are no links to any resources and the Table of Contents was blank. I was going to send this to someone interested in starting a shipping co-op in Alaska as an alternative to Amazon, but after looking at the website it seems like there is no actual info. Am I missing something?

**Comment by howtostartacoop** [comment]
Are you on mobile by chance?

**Comment by dhasial** [comment]
No, I'm on desktop. For example, clicking on anything just takes me back to the same page I'm already on. Same kind of thing with the table of contents, it only has one link, back to the home page. And I see the little blurb saying that the site has information collated from different sites, but I can't find such a collation. I took a couple of screenshots that might help explain what I'm seeing. (sorry for the delay in responding btw).
![howtostartacoop1/2](https://user-images.githubusercontent.com/44693166/106924393-eedf2c00-6706-11eb-86cf-ec469b5f67d4.png)
![howtostartacoop2/2](https://user-images.githubusercontent.com/44693166/106924404-f1418600-6706-11eb-9342-28f9371589ff.png)

**Comment by howtostartacoop** [comment]
That seems to be an issue upstream, desktop windows that are too narrow to not have the sidebar, but not narrow enough to have the mobile site do not have a functioning bar.
